common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -149,6 +149,17 @@ class InheritanceMixin(XBlockMixin):
         default=True,
         scope=Scope.settings
     )
+    video_bumper = Dict(
+        display_name=_("Video Bumper Settings"),
+        help=_(
+            """Enter video bumper settings in the following format:"""
```

Comment:
@lamagnifica Transcript file is not mandatory. Let's rephrase a bit? Regarding ua, this is typo, thanks for catching, should be uk. Updated description.